yarn.lock

```diff
@@ -3846,7 +3846,7 @@ braces@^3.0.1, braces@~3.0.2:
   dependencies:
     fill-range "^7.0.1"
 
-broadcast-channel@^4.10.0, broadcast-channel@^4.13.0:
+broadcast-channel@^4.10.0:
```

Comment:
Why is broadcast-channel `^4.10.0` still required?

Reply:
`@okta/okta-signin-widget ^6.4.1` is used for sample `react-embedded-auth-with-sdk`. It has `@okta/okta-auth-js 6.5.1` in dependencies which require `broadcast-channel ^4.10.0`

Reply:
This will be fixed on the widget side, then we can update widget version in auth-js. OK to leave this untouched in this PR